bundles/org.openhab.binding.qbus/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,266 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="qbus"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="bridge">
+		<label>Qbus Bridge</label>
+		<description>This bridge represents a Qbus client</description>
+		<config-description>
+			<parameter name="addr" type="text" required="true">
+				<label>IP or Host Name</label>
+				<description>IP Address of Qbus server, usually 'localhost'</description>
+				<default>localhost</default>
+				<advanced>false</advanced>
+				<context>network-address</context>
+			</parameter>
+			<parameter name="sn" type="text" required="true">
+				<label>Serial nr</label>
+				<description>Serial nr of the CTD controller</description>
+				<default></default>
+				<advanced>false</advanced>
+			</parameter>
+			<parameter name="port" type="integer" required="false">
+				<label>Bridge port</label>
+				<description>Port to communicate with Qbus server, default 8447</description>
+				<default>8447</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refresh" type="integer" required="false">
+				<label>Refresh interval</label>
+				<description>Refresh interval for connection with Qbus server (min), default 300. If set to 0 or left empty, no
+					refresh will be scheduled</description>
+				<default>300</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<thing-type id="onOff">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>Switch</label>
+		<description>Bistabiel-Mono-Timer-Interval output</description>
+		<channels>
+			<channel id="switch" typeId="switch"/>
```

Comment:
There are some predefined channel types. Then, you don't need to define your own. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types
```suggestion
			<channel id="switch" typeId="system.power"/>
```